.ci/pitest.sh

```diff
@@ -143,6 +143,8 @@ pitest-javadoc)
   "AbstractTypeAwareCheck.java.html:<td class='covered'><pre><span  class='survived'>            if (dotIdx == -1) {</span></pre></td></tr>"
   "AbstractTypeAwareCheck.java.html:<td class='covered'><pre><span  class='survived'>        imports.clear();</span></pre></td></tr>"
   "AbstractTypeAwareCheck.java.html:<td class='covered'><pre><span  class='survived'>        typeParams.clear();</span></pre></td></tr>"
+  "AllowedAnnotationsUtil.java.html:<td class='uncovered'><pre><span  class=''>    }</span></pre></td></tr>"
+  "AllowedAnnotationsUtil.java.html:<td class='uncovered'><pre><span  class=''>    private AllowedAnnotationsUtil() {</span></pre></td></tr>"
```

Comment:
this is not OK.
ALL new lines should be covered. this ignore list is unfortunate technical dept that we have, one day we will resole it

Reply:
I see, but this is a private constructor. It's not possible to invoke it, right? I added it to indicate it as a utility class ( HideUtilityClassConstructor in checkstyle rules ). 

I'll try to get rid of this class altogether based on the other comments